### DISCUSSION TITLE
How to edit public facing pages?

### DISCUSSION THREAD

**Discussion by VertyyBird** [ArchiveBox/ArchiveBox#1532] [Q&A]
I have my ArchiveBox set so the public can see my archive and submit URLs to be archived. I'd like to edit the public facing pages for some customisation and to remove certain things. I don't want to remove the attribution, but I want to remove the display of certain archive methods that I turned off in `ArchiveBox.conf`.

I've read the wiki but couldn't find anything on this besides a way to edit the footer.

I also looked at the code on this repo and see that there are template files for the public pages, but I have no idea how to get to them on my instance.

**Comment by pirate** [comment]
Have you seen these docs? https://github.com/ArchiveBox/ArchiveBox/wiki/Configuration#custom_templates_dir
